### PR TITLE
Document filter by star-rating and membership-access

### PR DESCRIPTION
**Pull request by oilnam**

### CODE REVIEW

File: documentation/md/content_search.md [hunk 1]
```diff
@@ -67,6 +67,8 @@ Name  | Description | Type | Accepted values | Boolean operators
 `ids` | Return only content with those IDs | *String* | e.g. technology/2014/feb/17/flappy-bird-clones-apple-google | false
 `production-office` | Return only content from those production offices | *String* | e.g. aus | true
 `lang` | Return only content in those languages | *String* | ISO language codes, e.g. en, fr | true
+`star-rating` | Return only content with a given star rating | *Integer* | 1 to 5 | false
+`membership-access` | Return only content with a given membership access value | *String* | members-only, paid-members-only | true
```
Comment (LATaylor-guardian) [inline documentation/md/content_search.md]: Do we want to be publicising this filter (membership-access)?

Comment (oilnam) [inline documentation/md/content_search.md]: Yeah good point. In hindsight, probably not? I'll ask @chriswilk . I think is safe to leave the star rating though?

Comment (LATaylor-guardian) [inline documentation/md/content_search.md]: Star rating is fine! :)

**Comment by LATaylor-guardian** [comment]
:+1:

**Pull request merged**